### PR TITLE
Create arachnologische-mitteilungen.csl

### PR DESCRIPTION
**Pull request by flrbrasil**
Citation style for Arachnologische Mitteilungen (Arachnology Letters): https://arages.de/en/journal/arachnology-letters

### CODE REVIEW

File: Aramit2023.csl [hunk 1]
```diff
@@ -0,0 +1,252 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="de">
+  <info>
+    <title>Arachnologische Mitteilungen</title>
+    <title-short>AraMit</title-short>
+    <id>http://csl.mendeley.com/styles/styles/Aramit2023</id>
```
Comment (POBrien333) [inline Aramit2023.csl:6]: change to: `<link href="http://www.zotero.org/styles/arachnologische-mitteilungen" rel="self"/>`

Comment (flrbrasil) [reply Aramit2023.csl]: How can I edit the submitted code?

Comment (flrbrasil) [reply Aramit2023.csl]: Found it!

**Comment by flrbrasil** [comment]
Is there anything I have to do that the style ist uploaded to the repository? I do not see any problems left but nothing happens. could you please explain me what I have to do now?

**Comment by POBrien333** [comment]
The PRs (currently there are 74) are worked through in reverse chronological order.
You'll have to wait a bit more until the maintainers review the January submissions. :)

**Comment by flrbrasil** [comment]
Ah, ok. No Problem. If I just have to wait that´s fine. I wanted to know if I did miss anything in my responsability. Thanks for the information.

**Comment by flrbrasil** [comment]
> Ah, ok. No Problem. If I just have to wait that´s fine. I wanted to know if I did miss anything in my responsability. Thanks for the information.

**Comment by flrbrasil** [comment]
> The PRs (currently there are 74) are worked through in reverse chronological order. You'll have to wait a bit more until the maintainers review the January submissions. :)

Just wanted to know when the January submissions will reviewed?

**Comment by adam3smith** [comment]
Sorry for the long wait -- please look through my edits above for future reference:
- avoid cluttering the style with default format info such as `font-style="normal"`
- strongly prefer using terms over hardcoded prefixes/suffixes
- if the style uses English terms (such as pp. and eds. rather than S. and Hrsg.), its default-locale should be set to English

**Comment by flrbrasil** [comment]
OK. Thank you very much for the valuable help and the helpful edits.

**Pull request merged**